src/test/java/org/zalando/intellij/swagger/fixture/SwaggerFixture.java

```diff
@@ -51,7 +52,7 @@ public void tearDown() throws Exception {
   @NotNull
   private AssertableList getCompletions(@NotNull String caretFileName) {
     List<String> results = myCodeInsightFixture.getCompletionVariants(caretFileName);
-    Assert.assertThat(results, IsNull.notNullValue());
+    assertThat(results, IsNull.notNullValue());
```

Comment:
Isn't the `isNotNull` matcher in the Hamcrest's `Matchers` summary class better to use?

Reply:
Thanks, changed.